Main.java

```diff
@@ -0,0 +1,62 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+
+        Scanner reader = new Scanner(System.in);
+
+        Circle myCircle = new Circle(0);
+        Rectangular myRectangular = new Rectangular(0, 0);
+        Triangle myTriangle = new Triangle(0, 0, 0);
+
+        ArrayList<Shape> shape = new ArrayList<>();
+        int answer;
+
+        do {
+            System.out.println("Perimeter and area calculation menu! Select the option you want: ");
+            String[] menuOptions = {
+                    "1.For a circle",
+                    "2.For a rectangular",
+                    "3.For a triangle",};
+            for (String menuElements : menuOptions) {
+                System.out.println(menuElements);
+            }
+            answer = reader.nextInt();
+
+            switch (answer) {
+                case 1:
+                    myCircle.addRadius();
```

Comment:
In all case statements there is one problem. I will illustrate it here:
```
myCircle.addRadius();
shape.add(new Circle(myCircle.getRadius()));
```

Basically, you are adding radius to object `myCircle` but then in the statement below, you are creating new Circle object and passing to the constructor value from the other object.
While this will technically work, it is not clean and optimised.

You can do this:
```
Circle myCircle = new Circle(); # pass value for the radius
myCircle.addRadius();
shape.add(myCircle);
```